jep/0000/README.adoc

```diff
@@ -0,0 +1,351 @@
+= JEP-0000: Java 10 and 11 support in Jenkins
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+
+.Metadata
+[cols="2"]
+|===
+| JEP
+| 0000
+
+| Title
+| Java 10 and 11 support in Jenkins
+
+| Sponsor
+| link:https://github.com/oleg-nenashev[Oleg Nenashev]
+
+// Use the script `set-jep-status <jep-number> <status>` to update the status.
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Process
+
+| Created
+| 2018-07-05
+
+| BDFL-Delegate
+| :bulb: Link to github user page :bulb:
```

Comment:
Do you have any thoughts on who should be the BDFL-Delegate for this plan?

Reply:
Options: 

* @kohsuke as BDFL and an initiator of the hackathon
* @daniel-beck / @olivergondza as weekly/LTS maintainers
* @dwnusbaum / @ndeloof / @svanoort as active contributors

But generally I would expect the most of the discussion to happen within the Platform SiG (#135), which could assign a BDFL delegate